packages/cwp-template-aws/cli/info/index.js

```diff
@@ -4,12 +4,19 @@ const { green } = require("chalk");
 const line = `-------------------------`;
 
 const printEnvOutput = async (env, context) => {
+    let output = await getStackOutput("api", env);
+
     console.log(line);
-    console.log(`Environment: ${green(env)}`);
+    console.log(
+        [
+            `Environment: ${green(env)}`,
+            `AWS region: ${green(output.region ?? "not deployed yet")}`,
+            `Project name: ${green(context.project.name)}`
+        ].join("\n")
+    );
     console.log(line);
 
     let stacksDeployedCount = 0;
-    let output = await getStackOutput("api", env);
     if (output) {
```

Comment:
@doitadrian Headless CMS URLs are showing undefined because we are checking for output that is an empty object. and an empty object evaluates to true in JS if statement.

we can use lodash isEmpty method to fix it.

Reply:
I'd just go with a label like "Not Available" when the CDN URL is not defined. I think that would make sense. We can try and see how it looks maybe.